roles/cluster_template/tasks/main.yaml

```diff
@@ -0,0 +1,176 @@
+- name: Create common dirs
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: '0755'
+  loop:
+    - "output/{{ clusterName }}"
+    - "output/{{ clusterName }}/feature/odf"
+    - "output/{{ clusterName }}/feature/odf/externalsecrets"
+    - "output/{{ clusterName }}/feature/odf/storageclasses"
+    - "output/{{ clusterName }}/feature/odf/redhatcop.redhat.io"
+    - "output/{{ clusterName }}/externalsecrets"
+    - "output/{{ clusterName }}/nodenetworkconfigurationpolicies"
+
+#Create dirs only found in external cluster
```

Comment:
"external" may be the wrong word here. We have effectively two axes we can use to describe clusters:

- `external`/`internal` -- this describes how the cluster is accessed. "internal" clusters require VPN access; "external" clusters have a public ip address.

- `user`/`admin` -- this describes how we expect the cluster to be *used*. On a "user" cluster we permit non-admin access and people can run arbitrary workloads in their namespaces. An "admin" cluster only allows logins from the NERC team.

So e.g. the observability cluster is an "external admin" cluster, while the test cluster is effectively an "internal user" cluster.